lib/network/modules/components/Node.js

```diff
@@ -245,15 +247,16 @@ class Node {
    * @static
    */
   static parseOptions(parentOptions, newOptions, allowDeletion = false, globalOptions = {}, groupList) {
-
     var fields = [
       'color',
+      'opacity',
       'fixed',
       'shadow'
     ];
     util.selectiveNotDeepExtend(fields, parentOptions, newOptions, allowDeletion);
 
     Node.checkMass(newOptions);
+    parentOptions.opacity = newOptions.opacity || parentOptions.opacity || 1;
```

Comment:
Do NOT modify group options from here! This way you'll set this nodes opacity to the group. Instead the group's opacity should be copied into the node if it doesn't have it's own.